arviz/tests/helpers.py

```diff
@@ -571,9 +571,14 @@ def pystan_version():
     """
     try:
         import pystan  # pylint: disable=import-error
+        version = int(pystan.__version__[0])
     except ImportError:
-        import stan as pystan  # pylint: disable=import-error
-    return int(pystan.__version__[0])
+        try:
+            import stan as pystan  # pylint: disable=import-error

```

Comment:
Curious, why is this extra block of logic needed?

Reply:
@canyon289 In the event that neither `pystan` or `stan` are installed, there would be no imported `pystan`, which would then have no version attribute. I believe that this function is called via `test_data_pystan.py`, and will throw an error just while collecting the tests. Even though this file would eventually be skipped because neither libraries are installed via:
```python
pytestmark = pytest.mark.skipif(
    ~pystan_installed, reason="test requires stan/pystan which is not installed"
)
```

This was my way to handle potentially two exceptions, and also indicate that there is no form of `stan` installed by returning `None`. (I have neither `stan`/`pystan` package, so this was a solution that I could get to work for me)

Reply:
Yeah, this sounds fine, maybe this could be `import stan` --> `stan.__version__`

Reply:
Hi @ahartikainen is that just regarding line 580? Still try to import `pystan`, and if not present try to import `stan`, but no need to `import stan as pystan`?